stage/services/salesforce-api/salesforce-api_runner.js

```diff
@@ -160,9 +186,9 @@ exports.handler = async function (event, context) {
 
     // some member types support billing information
     if (
-      ["Direct Member", "Consortium Organization", "Member Only"].includes(
-        res.attributes.member_type
-      )
+      ["Direct Member", "Consortium Organization", "Member Only"].includes(res.attributes.member_type) &&
+      hasBillingInfo(res) &&
+      hasJoinedDate(res)
```

Comment:
So what's the reasoning we dont want to send billing information if there is no date joined?

Reply:
We only want to send billing info if:

- There is a date joined.
- There is billing info in the update.